### ISSUE TITLE
T-Api in Python bindings

### ISSUE THREAD

**Issue by opencv-pushbot** [opencv/opencv#5043]
Transferred from http://code.opencv.org/issues/4443
```
|| Sander Eikelenboom on 2015-06-27 13:46
|| Priority: Normal
|| Affected: None
|| Category: None
|| Tracker: Feature
|| Difficulty: 
|| PR: 
|| Platform: None / None
```

T-Api in Python bindings
-----------
```
At present the python bindings don't seem to make use of UMat (and therefor openCL), it would be nice if the python bindings became on par again with C / C++.
```

History
-------

**Comment by Arqu** [comment]
Any progress on this? Can I help somehow, I'm very interested for bringing this to Python.

**Comment by TSC21** [comment]
Hi there! I'm really interested on this feature. Any updates on the state of this? Thanks!

**Comment by TSC21** [comment]
@mshabunin, alalek ping! Any updates on this? Thanks!

**Comment by xanview** [comment]
+1

**Comment by urvineet** [comment]
waiting for update

**Comment by alalek** [comment]
Take a look on this patch: #6078

**Comment by Arqu** [comment]
You, sir, are a true hero :) I guess this can be closed now.

**Comment by zerhacken** [comment]
Would love to see OpenCL support when using python bindings, any news in this area?

**Comment by paroj** [comment]
fixed by #6078
